tests/blockchain/eth/contracts/base/test_issuer.py

```diff
@@ -78,6 +78,11 @@ def calculate_reward(locked, total_locked, locked_periods):
     tx = token.functions.approve(issuer.address, economics.erc20_reward_supply).transact({'from': creator})
     testerchain.wait_for_receipt(tx)
 
+    # Can't burn tokens before initialization
+    with pytest.raises((TransactionFailed, ValueError)):
```

Comment:
Outside the scope of this PR, but it will read nice to set a variable for the exceptions tuple ie `TransactionFailed = (TransactionFailed, ValueError)`

Reply:
Yes! Perhaps not reusing the same name `TransactionFailed`, but another thing like `TransactionError`.